### DISCUSSION TITLE
안심번호 관련

### DISCUSSION THREAD

**Discussion by psh2849** [euphony-io/check-in-speaker#6] [General]
@euphony-io/euphoria 안심번호와 관련된 글은 여기서 토의하면 좋을 것 같습니다!

프로젝트를 진행하면서 안심번호를 어떻게 발급하고 관리할지가 중요하다고 느꼈습니다,,
어떻게 하면 효율적으로 유저들의 안심번호를 관리할 수 있을지 의견을 제시해주시면 좋을 것 같습니다! 

**EX) 안심번호 발급 방법 / 저장 방법 / 중복 체크 등등..**

**Comment by psh2849** [comment]
고유한 키값을 생성할 수 있는 방법을 찾아보던 중 `UUID` 라는 걸 알게 되었습니다.
일단 장점은 생성된 키들의 중복 확률이 매우 낮다는 점..? 

키 형식은 8-4-4-4-12 형식으로
`123e4567-e89b-12d3-a456-426655440000`
이런식으로 생성된다고 하네요. 

`var random_uuid = UUID.randomUUID()`
이런 식으로 한줄만에 생성 가능하다는 장점도 있습니다.. ㅎㅎ
생성된 값을 토대로 조금씩 변형을 줘도 좋을 것 같습니다.

참고했던 링크 공유할게요!
https://kkh0977.tistory.com/660
https://hojongs.github.io/uuid-definition/

**Reply by jopopcorn** [reply]
오오 아이디어는 좋은데 전화번호 11자리를 통째로 uuid로 변환하면 너무 길어질 것 같네요 ㅜ
말씀하신 것처럼 조금 변형할 수 있는 방법을 생각해보는 게 좋을 것 같아요!! 좋은 정보 감사드려요!

**Reply by psh2849** [reply]
앗.. 저는 그냥 uuid 하나만 생성하고 생성된걸 토대로 조금 변형을 주면 좋지 않을까.. 생각해봤습니다.. ㅎㅎ

**Reply by jopopcorn** [reply]
ㅋㅋㅋㅋㅋ앗 넵넵 다가올 화상회의 때 팀원분들과 최대한 상의해서 결정하는걸로 해요!
